snscrape/modules/twitter.py

```diff
@@ -1081,7 +1081,12 @@ def _make_tweet(self, tweet, user, retweetedTweet = None, quotedTweet = None, ca
 						card.url = u.url
 						break
 				else:
-					_logger.warning(f'Could not translate t.co card URL on tweet {tweetId}')
+					try:
+						u = self._head(card.url)
```

Comment:
Won't this slow down the scraper significantly?

Reply:
> Won't this slow down the scraper significantly?

It would  but I don't think it'd matter much. HEAD requests are very fast. Would it be better if this were opt-in?
Most tweets I scraped could have the t.co URL translated fine, I didn't hit the warning often.

Reply:
This is probably the big thing to figure out here first. (I'll have other remarks later.)

It should definitely be configurable. I'm not sure whether it should be on or off by default, though I'm leaning towards off (i.e. opt-in).

Reply:
Maybe the developer can shed some more light.